vowpalwabbit/parser_helper.h

```diff
@@ -32,9 +32,18 @@ class arguments {
   vw* all;//backdoor that should go away over time.
 
   //initialization
- arguments(vw& all_in, std::string name_in=""):new_od(name_in), missing_critical(false), all(&all_in) {file_options = new std::stringstream;};
- arguments():missing_critical(false){};//this should not be used but appears sometimes unavoidable.  Do an in-place allocation with the upper initializer after it is used.
-  ~arguments(){ delete file_options;};
+ arguments(vw& all_in, std::string name_in=""):new_od(name_in), missing_critical(false), all(&all_in) {
+   file_options = new std::stringstream;
+ };
+ arguments():missing_critical(false){
```

Comment:
Now that the default constructor is no longer one line, please reposition the comment.